examples/samg55_xplained_pro/usbserial/main.cpp

```diff
@@ -0,0 +1,31 @@
+
+#include "modm/platform.hpp"
+
+#include "modm/board.hpp"
```

Comment:
```suggestion
#include <modm/board.hpp>
```

to indicate library vs local files. Just a convention though.